binancial/data/get_trades_realtime.py

```
@@ -1,45 +1,95 @@
-def get_trades_realtime(stream, symbol, file_path):
+from typing import Any
+from datetime import datetime
+import logging
+import nest_asyncio
+from contextlib import contextmanager
+from pathlib import Path
 
+def get_trades_realtime(stream: Any, symbol: str, file_path: str | Path) -> None:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `stream`

(ANN401)